src/librustdoc/html/render.rs

```diff
@@ -1674,6 +1674,7 @@ impl<'a> Settings<'a> {
                 ("item-attributes", "Auto-hide item attributes.", true),
                 ("trait-implementations", "Auto-hide trait implementations documentation",
                  true),
+                ("method-docs", "Auto-hide item methods' documentation", false),
```

Comment:
Was this setting requested by anyone? It doesn't seem all that necessary to me. "Collapse all" is already saved into local storage.

Reply:
We were doing it before through "trait-implementations". Now that I've fixed it, I assume some people might want all their docs collapsed by default so why not providing it?